website/docs/deployment.mdx

```diff
@@ -473,12 +473,12 @@ A cross-repo publish is more difficult to set up because you need to push to ano
 3. Copy the key to clipboard with `pbcopy < ~/.ssh/id_rsa.pub` and paste it as a [deploy key](https://developer.github.com/v3/guides/managing-deploy-keys/#deploy-keys) in the deployment repository. Copy the file content if the command line doesn't work for you. Check the box for `Allow write access` before saving your deployment key.
 4. You'll need your private key as a [GitHub secret](https://help.github.com/en/actions/configuring-and-managing-workflows/creating-and-storing-encrypted-secrets) to allow Docusaurus to run the deployment for you.
 5. Copy your private key with `pbcopy < ~/.ssh/id_rsa` and paste a GitHub secret with the name `GH_PAGES_DEPLOY` on your source repository. Copy the file content if the command line doesn't work for you. Save your secret.
-6. Create your [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example it's `deploy.yml`.
+6. Create your [documentation workflow file](https://help.github.com/en/actions/configuring-and-managing-workflows/configuring-a-workflow#creating-a-workflow-file) in `.github/workflows/`. In this example, it's `deploy.yml`.
 
 At this point, you should have:
 
 - the source repo with the GitHub workflow set with the private SSH key as the GitHub Secret, and
-- your deployment repo set with the public SSH key in GitHub Deploy Keys.
+- your deployment repo is set with the public SSH key in GitHub Deploy Keys.
```

Comment:
This is a part of the sentence "At this point, you should have..." so it should be a noun, not a sentence, like the previous bullet point.